taskchampion/lib/src/kv.rs

```diff
@@ -100,6 +120,17 @@ impl CList for TCKVList {
     }
 }
 
+impl Default for TCKVList {
+    fn default() -> Self {
+        let (ptr, len, capacity) = vec_into_raw_parts(Vec::default());
+        Self {
+            items: ptr,
+            len,
+            _capacity: capacity,
+        }
+    }
```

Comment:
Preferring to use the PassByValue types:
```suggestion
        // SAFETY:
        //  - caller will free this list
        unsafe { TCKVList::return_val(Vec::new()) }
```